app/views/reports/regions/diabetes.html.erb

```diff
@@ -1,3 +1,4 @@
+<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0/dist/Chart.min.js" integrity="sha256-Uv9BNBucvCPipKQ2NS9wYpJmi8DTOEfTA/nH2aoJALw=" crossorigin="anonymous"></script>
```

Comment:
We are using a plugin called `ChartDataLabel`. For this plugin to work we have to load the JS script first.